pravega/flink-connectors
# [Issue-145] Add pravega client dependency to connector POM

Signed-off-by: zhongle.wang <zhongle_wang@dell.com>

**Change log description**

Remove the manual pom dependencies injections so that the created pom will have the right dependencies.

**Purpose of the change**

Fixes #145 

**What the code does**

* Remove the manual pom dependencies injections.
* Also remove the `bintray.gradle` that nobody uses.

**How to verify it**

`gradle install` will generate pom with right dependencies mentioned in the `build.gradle`.
This should also affect the `publishToRepo` task.
